rfc/rfc014-authorization-credential.md

```diff
@@ -0,0 +1,218 @@
+
+## 4. Access Control
+
+A resource server uses the Nuts Authorization Credential to check the access rights of an actor.
+The rules for determining access are a combination of a Bolt specific policy and any additional information from the credential.
+Identification and authentication are covered by [RFC003](rfc003-oauth2-authorization.md).
+If no `restrictions` are added to the credential, the resource server MUST follow the policy rules of the Bolt.
+The Bolt policy will list the operations and resource types that can be accessed. 
+A policy MAY also require certain parameters. For example, when a `search` operation is done on a FHIR `observation` resource, the policy may have a rule that requires a query parameter using the `subject` field of the credential.
+All restrictions and policy rules MUST use paths relative to the endpoint from the Nuts registry for the given service.
+If `restrictions` are present in the credential, the resource server can compare the operation and relative path of the request to the `restrictions` present in the credential.
+
+Although Nuts Authorization Credentials are part of the OAuth flow of [RFC003](rfc003-oauth2-authorization.md), the actual checking is done at request time. This means that the resource server will have to check the policy and make a request for the restrictions from the Nuts registry. This model can be compared with an Attribute Based Access Control (ABAC) model. The Bolt policies are added to the Policy Administration Point, the Nuts node acts as Policy Information Point. The resource server is the Policy Enforcement Point. It's up to the vendor to implement the Policy Decision Point. 
+
+## 5. Issuance & distribution
+
+A Nuts Authorization Credential is private and the transaction MAY be published over the Nuts network.
+The contents of the Credential MUST NOT be attached to the network transaction.
+Only the custodian and actor MAY request and retrieve the transaction payload.
+Every DID MAY issue an authorization credential.
+The VC does not have any other requirements nor does it add requirements to other VCs.
+
+## 6. Supported proofs
+
+Only proofs from [RFC011](rfc011-verifiable-credential.md) are supported.
+
+## 7. Trust
+
+The Nuts Authorization Credential only impacts the actor and custodian.
+It MUST be trusted automatically.
+
+## 8. Revocation
+
+The Nuts Authorization Credential uses the revocation mechanism as stated by [RFC011](rfc011-verifiable-credential.md).
+
+## 9. Use cases
+
+The Nuts Authorization Credential MUST be used in the Nuts OAuth flow as stated in [RFC003](rfc003-oauth2-authorization.md).
```

Comment:
As consent, as referenced in scetion 3.2 of RFC003? If so, I think RFC003 should be expanded on that part because the explanation is now very, very brief.

Reply:
There are also additions in 4.2.2, 5.2.1.7 and 6.2, point 2